src/util/invariant.h

```diff
@@ -142,11 +142,6 @@ class invariant_with_diagnostics_failedt : public invariant_failedt
 #else
 #define CBMC_NORETURN
 #endif
-// The macros MACRO<n> (e.g., INVARIANT2) are for internal use in this file
-// only. The corresponding macros that take a variable number of arguments (see
-// further below) should be used instead, which in turn call those with a fixed
-// number of arguments. For example, if INVARIANT(...) is called with two
-// arguments, it calls INVARIANT2().
 
 #if defined(CPROVER_INVARIANT_CPROVER_ASSERT)
```

Comment:
In the following we should also add the new macro `INVARIANT_WITH_DIAGNOSTICS`, etc. and remove the macros `INVARIANT3`, etc.

Reply:
Done, thanks!